web/src/pages/profile.tsx

```diff
@@ -36,7 +36,7 @@ import { IsAuthenticated } from "@/lib/auth/AuthContext";
 import { useConfig } from "@/lib/data-hooks/useConfig";
 import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
 import { useUserData } from "@/lib/data-hooks/useUserData";
-import { isHomeBasedBusinessApplicable } from "@/lib/domain-logic/isHomeBasedBusinessApplicable";
+import { isHomeBasedBusinessApplicable } from "@/lib/domain-logic/essentialQuestions/isHomeBasedBusinessApplicable";
```

Comment:
I think this file should probably be moved out of this folder given that homeBasedBusiness isn't really an essential industry-based question